help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Acxiom Audience Distribution
+description: Use the [!DNL Acxiom Audience Distribution] destination to enhance audiences with [!DNL Acxiom's Real ID] technology and activate audiences to multiple platforms, such as [!DNL Altice], [!DNL Ampersand], [!DNL Comcast], and more.
+badge: Beta
+---
+# [!DNL Acxiom Audience Distribution] destination 
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Audience Distribution] destination is in beta. This destination connector and documentation page are created and maintained by the [!DNL Acxiom] team. For any inquiries or update requests, contact Acxiom directly [here](mailto:acxiom-adobe-help@acxiom.com).
+
+Use the [!DNL Acxiom Audience Distribution] destination to enhance audiences with [!DNL Acxiom's Real ID] technology and activate audiences to multiple platforms, such as [!DNL Altice], [!DNL Ampersand], [!DNL Comcast], and more.
+
+This tutorial provides instructions to create an [!DNL Acxiom Audience Distribution] destination connector using the [!DNL Adobe Experience Platform] user interface. This connector is used to build and distribute audiences to selected destinations.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Acxiom Audience Distribution] destination, here is a sample use case that [!DNL Adobe Experience Platform] customers can solve by using this connector.
+
+### Send audiences from Experience Platform to your Acxiom account {#send-audiences}
+
+Use this destination connector if you are a marketing professional who wants to send audiences from [!DNL Experience Platform] to your [!DNL Acxiom] account, for cross-channel acquisition.
+
+For example, the Marketing Operations department at a global financial services brand uses the [!DNL Acxiom Audience Distribution] destination to send audiences from [!DNL Experience Platform] to [!DNL Acxiom] for cross-channel acquisition.
+
+
+While the segmentation runs, you must set up the connection to a specific destination (e.g., [!DNL LG Ads], [!DNL Spectrum], [!DNL Altrice], etc.).

```

Comment:
These last two sentences are not use-case specific info. These are just descriptions of Platform functionality. Let's rephrase this as such:

For example, the Marketing Operations department at a global financial services brand is interested in cross-channel customer acquisition through multiple advertising platforms. They can use the [!DNL Acxiom Audience Distribution] destination connector to send audiences from [!DNL Experience Platform] to [!DNL Acxiom], enhance the audiences with [!DNL Acxiom's Real ID] technology, and activate the audiences to multiple platforms, such as [!DNL Altice], [!DNL Ampersand], [!DNL Comcast], and more.


Please make sure the phrasing above is technically accurate.

Reply:
Done. Thank you!